src/pages/consent/consent-page.ts

```diff
@@ -31,27 +27,32 @@ export class ConsentPage extends LitElement {
   }
 
   render() {
+    preloadImage("./images/demo.gif");
```

Comment:
Do not put this in `render`. Put this in `firstUpdated`

Reply:
`render` is called on every render. FirstUpdated is only called once, but only after the component has rendered, so the user can interact with it.